flake.nix

```diff
@@ -241,18 +241,38 @@
         rec {
           inherit cargoArtifacts;
           nickel-lang-core = buildPackage { pname = "nickel-lang-core"; };
-          nickel-lang-cli = buildPackage { pname = "nickel-lang-cli"; };
+          nickel-lang-cli = buildPackage {
+            pname = "nickel-lang-cli";
+            extraBuildArgs = "--features format";
```

Comment:
If we want  format support by default, I think we should rather make it a default feature in `Cargo.toml`, to 1. avoid spreading feature configuration across Nix and Rust 2. have `cargo install` and `cargo build` produce the same binary as `nix build` by default.